R/fcm-methods.R

```diff
@@ -142,7 +142,7 @@ matrix2fcm <- function(x, meta = NULL) {
     if (is.null(meta))
         meta <- make_meta("fcm")
     
-    build_fcm(as(Matrix(x, sparse = TRUE), "dgCMatrix"),
+    build_fcm(as(as(Matrix(x, sparse = TRUE), "generalMatrix"), "dMatrix"),
```

Comment:
I don't think you need `as()` here because you added `as(as(as(x, "CsparseMatrix"), "generalMatrix"), "dMatrix")` to  `build_fcm()`.

Reply:
True. Hard to keep track of when it's needed. Was used before so I added it, but it wasn't necessary so I made it just 
```r
build_fcm(x, rowname, colname, meta = meta)
```
now.